packages/zoltan2/sphynx/src/Zoltan2_Sphynx.hpp

```diff
@@ -694,10 +694,10 @@ namespace Zoltan2 {
       Teuchos::ParameterList smootherParamList;
       smootherParamList.set("chebyshev: degree", 3);
       smootherParamList.set("chebyshev: ratio eigenvalue", 7.0);
-      smootherParamList.set("chebyshev: eigenvalue max iterations", 10);
+      smootherParamList.set("chebyshev: eigenvalue max iterations", irregular_ ? 100 : 10);
```

Comment:
Oh, are more iterations needed for irregular problems?

Reply:
Yes, I tried different combinations of 10 and 100 for smootherParamList.set("chebyshev: eigenvalue max iterations", x) and coarseParamList.set("chebyshev: eigenvalue max iterations", x), and concluded that this is the best combination.